composer.json

```
@@ -99,14 +99,14 @@
         "symfony/http-foundation": "4.4.*",
         "symfony/http-kernel": "4.4.*",
         "symfony/lock": "4.4.*",
+        "symfony/mailer": "^4.4 || 5.0.* || 5.1.*",
```

```suggestion
        "symfony/mailer": "4.4.*",
```

Because that is how you have requested the package in the `core-bundle/composer.json` file.

---

I have updated the dependencies now. Since Symfony 5.0 is EoL, I have changed it to `^4.4 || 5.1.*`. _Note:_ I also updated the `extra.symfony.require` accordingly. This is something we have to change for `contao/managed-edition` as well.